examples/js/controls/TransformControls.js

```diff
@@ -161,36 +165,45 @@
 					child.position.set( 0, 0, 0 );
 					child.rotation.set( 0, 0, 0 );
 					child.scale.set( 1, 1, 1 );
+
 				}
+
 			});
 
 		};
 
 		this.hide = function () {
-			this.traverse(function( child ) {
```

Comment:
I think this shouldn't be needed now, right? I think this code is there because, at the time, `WebGLRenderer` would render the object's children regardless of the parents `visible`.

So `this.visible = false` and `this.visible = true` should be all I think.